packages/contracts-bedrock/scripts/print-addresses.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# Default directory base path
+BASE_DIR="./deployments"
```

Comment:
Can you use the dir trick to get an absolute path from root no matter what `PWD` is? See this line: https://github.com/ethereum-optimism/optimism/blob/d893a531455deff27dd4e24e988029802d06eaf8/packages/contracts-bedrock/scripts/check-deploy-configs.sh#L9

Reply:
Done